xbmc/addons/kodi-addon-dev-kit/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+project(kodi-addon-dev-kit)
+
+cmake_minimum_required(VERSION 3.0.0)
```

Comment:
This has to be moved before project (as it set's some policies). Also please just use 3.1 (a patch version doesn't add new features). 3.1 because I'll propose to use CXX_STANDARD and we have the same for the main cmake buildsystem.